x.py

```diff
@@ -256,17 +288,25 @@ def test_go(dir: str, rest: List[str]) -> None:
 
     subparsers = parser.add_subparsers()
 
+    parser_format = subparsers.add_parser(
+        'format',
+        description="Format source code",
+        help="Format source code")
+    parser_format.set_defaults(func=lambda **args: check_format(**args, fix=True))
+    parser_format.add_argument('--clang-format-path', default='clang-format', help="path of clang-format used to check source")
+
     parser_check = subparsers.add_parser(
         'check',
         description="Check or lint source code",
         help="Check or lint source code")
     parser_check.set_defaults(func=parser_check.print_help)
     parser_check_subparsers = parser_check.add_subparsers()
-    parser_check_cpplint = parser_check_subparsers.add_parser(
-        'cpplint',
-        description="Lint code with cpplint (https://github.com/cpplint/cpplint)",
-        help="Lint code with cpplint (https://github.com/cpplint/cpplint)")
-    parser_check_cpplint.set_defaults(func=cpplint)
+    parser_check_format = parser_check_subparsers.add_parser(
+        'format',
+        description="Check source format by clang-tidy",
+        help="Check source format by clang-tidy")
```

Comment:
I think `clang-tidy` here should be `clang-format`

Reply:
done